player/player.cfg

```diff
@@ -130,10 +130,10 @@ iqmanim "pistol secondary|claw primary|sword primary" "claw_primary.iqm" 45
 iqmanim "claw secondary" "claw_secondary.iqm" 25
 iqmanim "sword secondary" "sword_secondary.iqm" 20
 
-if (|| [strcmp (mdlname) "actors/player/female/headless"] [strcmp (mdlname) "actors/player/male/headless"]) [
-    iqmskin * "<dds>headless.png" "<dds>mask.png" 0.25 0.0 1 0
+if (|| [strcmp (mdlname) "<dds>actors/player/female/headless"] [strcmp (mdlname) "<dds>actors/player/male/headless"]) [
```

Comment:
Not sure why you switched this here, but afaik it's incorrect.

Reply:
Ah, yep, I made a dumb.